docset/winserver2022-ps/adcsadministration/Confirm-CAAttestationIdentityKeyInfo.md

```diff
@@ -0,0 +1,115 @@
+---
+description: Checks whether the local certification authority trusts secure hardware for identity attestation.
+external help file: Microsoft.CertificateServices.Administration.Commands.dll-Help.xml
+Module Name: ADCSAdministration
+online version: https://docs.microsoft.com/powershell/module/adcsadministration/confirm-caattestationidentitykeyinfo?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Confirm-CAAttestationIdentityKeyInfo
+---
+
+# Confirm-CAAttestationIdentityKeyInfo
+
+## SYNOPSIS
+Checks whether the local certification authority trusts secure hardware for identity attestation.
+
+## SYNTAX
+
+### PublicKeyHash
+```
+Confirm-CAAttestationIdentityKeyInfo [-PublicKeyHash] <String> [<CommonParameters>]
+```
+
+### Certificate
+```
+Confirm-CAAttestationIdentityKeyInfo [-Certificate] <X509Certificate2> [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Confirm-CAAttestationIdentityKeyInfo** cmdlet checks whether the local certification authority (CA) trusts secure hardware, such as a Trusted Platform Module (TPM), for identity attestation. The Attestation Identity Key replaces the Endorsement Key as an identity for the TPM. An Attestation Identity Key is permanently embedded in the security hardware. The public portion of the key helps to recognize genuine security hardware.
+
+This cmdlet verifies whether the public certificate connects through a certificate chain to an anchor that the CA trusts. Specify an X509 certificate by using the **Certificate** parameter.
+
+This cmdlet checks whether the attestation identity public key exists as a file in a folder configured at the local CA for key attestation. Specify the public key by using the **PublicKeyHash** parameter.
```

Comment:
use capital letters for attestation identity - "... Attestation Identity Key (AIK) public key exists ..."